src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -165,9 +167,33 @@ public static StaffName parseStaffName(String name) throws ParseException {
     public static StaffTitle parseStaffTitle(String title) throws ParseException {
         requireNonNull(title);
         String trimmedTitle = title.trim();
-        if (!StaffTitle.isValidStaffTitle(title)) {
+        if (!StaffTitle.isValidStaffTitle(trimmedTitle)) {
```

Comment:
I like that there were changes to the variable names so that they were more descriptive.